src/NATS.Client.JetStream/Models/ConsumerCreateRequest.cs

```diff
@@ -26,4 +28,13 @@ internal record ConsumerCreateRequest
     [System.Text.Json.Serialization.JsonPropertyName("config")]
     [System.Text.Json.Serialization.JsonIgnore(Condition = System.Text.Json.Serialization.JsonIgnoreCondition.Never)]
     public ConsumerConfig? Config { get; set; }
+
+    [System.Text.Json.Serialization.JsonPropertyName("action")]
+    [System.Text.Json.Serialization.JsonIgnore(Condition = System.Text.Json.Serialization.JsonIgnoreCondition.Never)]
+#if NET6_0
+    [System.Text.Json.Serialization.JsonConverter(typeof(NatsJSJsonStringEnumConverter<ConsumerCreateRequestAction>))]
+#else
+    [System.Text.Json.Serialization.JsonConverter(typeof(NatsJSJsonStringEnumConverter<ConsumerCreateRequestAction>))]
+#endif
+    public ConsumerCreateRequestAction Action { get; set; }
```

Comment:
Could we make this optional so that we don't have to deal with net8 serialization exception for an empty value?

Reply:
Yes, that was one of the ideas. In this case, the property will sample be ignored and `NatsJSJsonStringEnumConverter` will be less strict